git.sh

```diff
@@ -0,0 +1,32 @@
+if [ -z $1 ]; then
+    exit 1
+fi
+if [ -z $2 ]; then
+    exit 1
+fi
+if [ -z $3 ]; then
+    exit 1
+fi
+if [ -z $4 ]; then
+    exit 1
+fi
```

Comment:
I'd add some error messages here before you exit. I mean all four parameters are required so a generic message like "Please provide FOUR parameters: `<folder> <branch> <suffix> <repository url>`"  will help if someone messes up.